sdk/search/azure-search-documents/src/test/java/com/azure/search/documents/indexes/CustomAnalyzerSyncTests.java

```diff
@@ -95,7 +96,6 @@
 
 public class CustomAnalyzerSyncTests extends SearchTestBase {
     private static final String NAME_PREFIX = "azsmnet";
-    private static final Collection<CharFilterName> CHAR_FILTER_NAMES = new ArrayList<>(CharFilterName.values());
```

Comment:
We have this because we added custom char names enum values for some tests. The Enum set changed during the other tests which fail the tests.

```
public void test1 () {
    CharFilterName testName = CharFilterName.fromString("test");
}

public void test2() {
     CharFilterName.values() // Probably have the "test" set by above test.
}
```